vdom/create-element.js

```diff
@@ -26,6 +26,7 @@ function createElement(vnode, opts) {
         return null
     }
 
+    var tagName = vnode.tagName || "span"
```

Comment:
How does span's minimality affect your library? span is inline and div is block, I'd guess div is the most neutral of all.